ui/src/map/layers/utils/switch-layer-utils.ts

```diff
@@ -14,33 +20,37 @@ import { findMatchingEntities, pointToCoords } from 'map/layers/utils/layer-util
 import { SearchItemsOptions } from 'map/layers/utils/layer-model';
 import VectorSource from 'ol/source/Vector';
 import { Rectangle } from 'model/geometry';
+import { ValidatedAsset } from 'publication/publication-model';
 
 const switchImage: HTMLImageElement = new Image();
 switchImage.src = `data:image/svg+xml;utf8,${encodeURIComponent(SwitchIcon)}`;
 
+const switchErrorImage: HTMLImageElement = new Image();
+switchErrorImage.src = `data:image/svg+xml;utf8,${encodeURIComponent(SwitchErrorIcon)}`;
+
 const TEXT_FONT_LARGE = 11;
 const TEXT_FONT_SMALL = 10;
 const CIRCLE_RADIUS_SMALL = 4.5;
 const CIRCLE_RADIUS_LARGE = 6.5;
 
 export function getSelectedSwitchLabelRenderer(
     layoutSwitch: LayoutSwitch,
-    large: boolean,
     linked: boolean,
+    isValid: boolean,
```

Comment:
Näistähän taidettiin vähän jutskailla jo eri pullarissa että käytetäänkö tuota is-prefixiä. Minusta tämä on aika tyypillistä, mutta Jyrkillä oli javataustaisena erilainen tulkinta. Varmaankin sama muoto kannattaisi pitää mikä ylemmässä "linked" lippuparametrissa (ehkäpä siis validated?).

Suosisin ehkäpä itse vielä pidempää muotoa, luokkaan switchValidationSucceeded tms, mutta mennee tämä näinkin.

Reply:
Näitä olikin vähän kaikkialla täällä tiedostossa. Yhteen funktioon vielä jätin näitä, sillä siellä a) kyseiset parametrit olivat funktioita (mielestäni on ihan ok, että jos predikaatti on nimetty is-alkuiseksi) ja b) ka. funktioissa oli samoilla nimillä myös lokaalit muuttujat, jotka saivat arvonsa ka. predikaateilta.